triage/2025-02-18.md

```diff
@@ -0,0 +1,147 @@
+| Improvements ✅ <br /> (secondary) | -0.4% | [-0.4%, -0.4%] | 2     |
+| All ❌✅ (primary)                 | 0.5%  | [0.5%, 0.5%]   | 1     |
+
+- Tiny regressions in doc builds, probably caused by more documentation in portable-simd.
+- Marked as triaged.
+
+transmute should also assume non-null pointers [#136735](https://github.com/rust-lang/rust/pull/136735) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=905b1bf1ccccaf091a880b069f80dc38ad9ecad3&end=d88ffcdb8bfc6f8b917574c1693eb9764a20eff5&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.5%  | [0.3%, 0.8%]   | 4     |
+| Regressions ❌ <br /> (secondary)  | 0.2%  | [0.2%, 0.2%]   | 1     |
+| Improvements ✅ <br /> (primary)   | -0.5% | [-1.0%, -0.2%] | 7     |
+| Improvements ✅ <br /> (secondary) | -0.2% | [-0.2%, -0.2%] | 2     |
+| All ❌✅ (primary)                 | -0.2% | [-1.0%, 0.8%]  | 11    |
+
+- Performance is a wash.
+- Marked as triaged.
+
+Rollup of 7 pull requests [#137163](https://github.com/rust-lang/rust/pull/137163) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=d5eb31c9347ae3c494c8d723711dacca7d1cfe8b&end=273465e1f2932a30a5b56ac95859cdc86f3f33fa&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | -     | -              | 0     |
+| Regressions ❌ <br /> (secondary)  | 0.1%  | [0.1%, 0.1%]   | 1     |
+| Improvements ✅ <br /> (primary)   | -0.3% | [-0.3%, -0.3%] | 2     |
+| Improvements ✅ <br /> (secondary) | -     | -              | 0     |
+| All ❌✅ (primary)                 | -0.3% | [-0.3%, -0.3%] | 2     |
+
+- Three tiny changes, the only regression is on an incr-unchanged run of a parser stress test,
+  probably doesn't warrant further investigation.
+- Marked as triaged.
+
+Update to LLVM 20 [#135763](https://github.com/rust-lang/rust/pull/135763) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=2162e9d4b18525e4eb542fed9985921276512d7c&end=ce36a966c79e109dabeef7a47fe68e5294c6d71e&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 5.1%  | [0.2%, 35.1%]  | 8     |
+| Regressions ❌ <br /> (secondary)  | 0.7%  | [0.4%, 1.0%]   | 10    |
+| Improvements ✅ <br /> (primary)   | -1.3% | [-4.2%, -0.2%] | 251   |
+| Improvements ✅ <br /> (secondary) | -1.0% | [-4.7%, -0.1%] | 178   |
+| All ❌✅ (primary)                 | -1.1% | [-4.2%, 35.1%] | 259   |
+
+- As usually, update to a new lLVM version brings a lot of performance wins, but also some regressions.
+- The single large regression was on a release (optimized) incremental build.
```

Comment:
Maybe more for twir’s sake than the compiler team: we can say that this is not a configuration used in practice by cargo but is still benchmarked?

Reply:
Done!